apps/services/handlers.py

```diff
@@ -138,14 +138,14 @@ async def create_user_request_review(
             data={
                 "university_id": university_id,
                 "user_request_id": user_request_id,
-                "date_created": datetime.now(),
+                "created_at": datetime.now(),
                 "reviewer": user.user_id,
                 "hostel_id": user_request_review.hostel_id,
                 "room_number": user_request_review.room_number,
-                "start_date_accommodation": user_request_review.start_date_accommodation.now(),
-                "end_date_accommodation": user_request_review.end_date_accommodation.now(),
+                "start_accommodation_date": user_request_review.start_accommodation_date.today(),
+                "end_accommodation_date": user_request_review.end_accommodation_date.today(),
                 "total_sum": user_request_review.total_sum,
-                "payment_deadline": user_request_review.payment_deadline.now(),
+                "payment_deadline_date": user_request_review.payment_deadline_date.today(),
```

Comment:
did you test it?

Reply:
Of course I'd tested it, everything worked fine.

Reply:
i fixed this issue